docs/src/main/asciidoc/kafka-support.adoc

```diff
@@ -1,7 +1,7 @@
 [#kafka-support]
 == Kafka Support
 
-Connect to Azure Event Hubs using Spring Kafka libraries. There are two approaches to connect to Azure Event Hubs for Kafka, the first one is to provide the Azure Event Hubs connection string directly, the other is to use Azure Resource Manager to retrieve the connection string.
+Connect to Azure Event Hubs(Basic pricing tier is not supported) using Spring Kafka libraries. There are two approaches to connect to Azure Event Hubs for Kafka, the first one is to provide the Azure Event Hubs connection string directly, the other is to use Azure Resource Manager to retrieve the connection string.
```

Comment:
add a space before `(`

Reply:
Maybe also add a link to the pricing tier description / support for Event Hubs here.